packages/cli/commands/theme/preview.js

```diff
@@ -0,0 +1,175 @@
+const fs = require('fs');
+const path = require('path');
+const { i18n } = require('../../lib/lang');
+const { logger } = require('@hubspot/cli-lib/logger');
+const {
+  addAccountOptions,
+  addConfigOptions,
+  getAccountId,
+} = require('../../lib/commonOpts');
+const { getCwd } = require('@hubspot/cli-lib/path');
+const { preview } = require('@hubspot/cli-lib/lib/preview');
+const { getUploadableFileList } = require('../../lib/upload');
+const { trackCommandUsage } = require('../../lib/usageTracking');
+const { loadAndValidateOptions } = require('../../lib/validation');
+const { previewPrompt } = require('../../lib/prompts/previewPrompt');
+const { EXIT_CODES } = require('../../lib/enums/exitCodes');
+const { FileUploadResultType } = require('@hubspot/cli-lib/lib/uploadFolder');
+const i18nKey = 'cli.commands.preview';
+const cliProgress = require('cli-progress');
+const {
+  ApiErrorContext,
+  logApiUploadErrorInstance,
+} = require('@hubspot/cli-lib/errorHandlers');
```

Comment:
If possible would like to add as few new imports to `cli-lib` as possible. For the new stuff you've added there its alright, but for anything else (`path` and `logger` for example) we should use the local-dev-lib version.\

For `errorHandlers`, these actually exist within the CLI now so can import them from `lib/errorHandlers`